grails-app/domain/edu/oswego/cs/lakerpolling/domains/User.groovy

```diff
@@ -11,9 +11,9 @@ class User {
     String imageUrl
 
     static hasOne = [authToken: AuthToken, role: Role]
+    static hasMany = [course: Course]
```

Comment:
After running some tests yesterday, I'd suggest you remove this line. Or perhaps, find another way to specify the two way relationship. This creates an error. The course having the hasMany on users is actually enough to be honest.